addons/mail/static/src/models/channel_command/channel_command.js

```diff
@@ -10,7 +10,7 @@ function factory(dependencies) {
 
     ChannelCommand.fields = {
         /**
-         * FIXME use this value task-2343850
+         * Makes only sense for threads on model 'mail.channel'. Type of the channel (e.g. 'chat', 'channel' or 'groups')
```

Comment:
I think this one is a bit long, should use new line somewhere.

Reply:
Also I would clarify it a bit: the "what" before the "exceptions" by adding ```Determines on which channel types `this` is available.``` before the current doc.